packages/Tooltip/package.json

```diff
@@ -22,6 +22,9 @@
   "files": [
     "dist"
   ],
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  },
```

Comment:
You should also update the template `utils/bootstrap/template/package.json` whith this script section for the future component bootstrap.

Reply:
Done